### ISSUE TITLE
Mandrel image fails to build native image with wasmtime-java (`GLIBC_2.32` not found)

### ISSUE THREAD

**Issue by cardil** [quarkusio/quarkus-images#227]
Hi. I'm trying to build the Quarkus app with the [wasmtime-java 0.14.0](https://github.com/kawamuray/wasmtime-java) library. The application works, but the native image build fails with:

```
/lib64/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/libwasmtime_jni_0.14.0_linux15928750139324800087.so)
```

I think that's because the Mandrel image used (default to `registry.access.redhat.com/quarkus/mandrel-22-rhel9:22.3-4`) is based on RHEL 8, and those contain glibc 2.28-189.5 ([see](https://catalog.redhat.com/software/containers/quarkus/mandrel-22-rhel8/63176245bee3c67f036b802c?container-tabs=packages)). It works well on my F37 machine. I have glibc 2.36-9.

Maybe we could have RHEL 9-based images as well. That would make this work.

**Comment by cescoffier** [comment]
We have the plan to switch to ubi:9 at some point in the future.

**Comment by cardil** [comment]
> We have the plan to switch to ubi:9 at some point in the future.

Yup. Makes sense. How about providing `ubi:8` and `ubi:9` images, so the transition be easier?

**Comment by cescoffier** [comment]
Yes, but we need to find a proper way to provide both (probably with ubi:9 in the image name or tag). Unfortunately it will make the maintenance a bit more cumbersome., and the migration annoying once we abandon ubi:8 and the tag gets reverted.

**Comment by rsvoboda** [comment]
@cescoffier what about this one?

**Comment by cescoffier** [comment]
Yes, closing this issue as out of date as we now deliver the UBI 9 (and soon 10) images.

Now, if you try to use something that requires a GLIBC version greater than the one provided by the image (regardless of the image), it won't work. So, the specific issue is fixed, but the mismatch may still happen.
